custom-routes-vis/src/main/java/com/google/sps/servlets/GetDirectionsServlet.java

```diff
@@ -48,19 +48,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     String requestApiKey = request.getParameter("apiKey");
     String directionsApiKey = System.getenv("DIRECTIONS_API_KEY");
-    // Use the Directions API key if no API key is specified.
-    String apiKey = requestApiKey.isEmpty() ? directionsApiKey : requestApiKey;
+    String customRoutesApiKey = System.getenv("CUSTOM_ROUTES_API_KEY");
```

Comment:
See my comment on read me. I don't think we should make the key Zihan allowed us to use a default key any user can use if they don't give one. I think they should be required to pass in the key through the website still.

Reply:
You can check on the chat the conversation I had with Zihan yesterday. I think this is what he meant